samples/XMasControlAddIn/AddIn/AddInInterface.al

```diff
@@ -0,0 +1,18 @@
+controladdin NavSnowControlAddIn

```

Comment:
I think AddInInterface.al should be named like the class itself, NavSnowControlAddIn.